File: Lenguaje de Programación 2/LP/Parcial/L3/Administrativo.cs
public class Administrativo : Persona, IConsultable{
	private string nombre;
	private string ap;
	private string am;
	private string regimen;
	private string funcion;
	
	public Administrativo(double salario, TipoDedicacion t, int anexo, string sexo, int dni,
	int edad,string estadoCivil,string nombre,string ap,string am,string regimen,string funcion){
		super( salario,  t,  anexo,sexo,  dni,  edad,estadoCivil);
		this.nombre=nombre;
		this.ap=ap;
		this.am=am;
		this.regimen=regimen;
		this.funcion=funcion;
	}
	
	public string consultarDatos(){
		TipoDedicacion tipo = getT();
		return  (nombre + " "+ap + " "+am + " "+ tipo + " "+regimen);
	}
}

File: Lenguaje de Programación 2/Labs/20151521 (2)/20151521/Administrativo.java
public class Administrativo extends Persona implements IConsultable{
	private String nombre;
	private String ap;
	private String am;
	private String regimen;
	private String funcion;
	
	public Administrativo(double salario, TipoDedicacion t, int anexo, String sexo, int dni,
	int edad,String estadoCivil,String nombre,String ap,String am,String regimen,String funcion){
		super( salario,  t,  anexo,sexo,  dni,  edad,estadoCivil);
		this.nombre=nombre;
		this.ap=ap;
		this.am=am;
		this.regimen=regimen;
		this.funcion=funcion;
	}
	
	public String consultarDatos(){
		TipoDedicacion tipo = getT();
		return  (nombre + " "+ap + " "+am + " "+ tipo + " "+regimen);
	}
}